src/reducers/cacheReducer.js

```diff
@@ -150,7 +150,16 @@ const fieldsTransducer = (fields) =>
 const orderTransducer = (order) => {
   const isFlat = typeof order[0] === 'string';
   const orders = isFlat ? [order] : order;
-  return partialRight.apply(null, [orderBy, ...zip.apply(null, orders)]);
+  const [fields, direction] = zip(
+    ...orders.map(([field, dir]) => [
+      (data) =>
+        typeof data[field] === 'string'
+          ? data[field].toLowerCase()
```

Comment:
Could you give an example of a situation when we need to make this field lowercase? I am trying to understand better what's going on :)

Reply:
After I implemented the optimistic reads the order would shift. 

The cause is that Firestore ordering is different than lodash's ordering. Change strings to lowercase ensures that results sorted by a string will come back the same order whether via Firestore's query syntax or the in-memory ordering done here.

Reply:
that's all thanks